src/main/java/com/microsoft/jenkins/appservice/commands/DockerDeployCommand.java

```diff
@@ -72,21 +81,7 @@ public void execute(final IDockerDeployCommandData context) {
                         slot.resourceGroupName(), webApp.name(), slotName);
                 checkNotNull(siteConfigResourceInner, "Configuration not found for slot:" + slotName);
 
-                List<NameValuePair> appSettings = new ArrayList<>();
-                appSettings.add(new NameValuePair()
-                        .withName(SETTING_DOCKER_IMAGE)
-                        .withValue(image));
-                appSettings.add(new NameValuePair()
-                        .withName(SETTING_REGISTRY_SERVER)
-                        .withValue(authConfig.getRegistryAddress()));
-                appSettings.add(new NameValuePair()
-                        .withName(SETTING_REGISTRY_USERNAME)
-                        .withValue(authConfig.getUsername()));
-                appSettings.add(new NameValuePair()
-                        .withName(SETTING_REGISTRY_PASSWORD)
-                        .withValue(authConfig.getPassword()));
                 siteConfigResourceInner.withLinuxFxVersion(String.format("DOCKER|%s", image));
-                siteConfigResourceInner.withAppSettings(appSettings);
```

Comment:
Looks like this no long works. Switch to another API to set application settings.